pkg/option/runtime_options.go

```diff
@@ -0,0 +1,102 @@
+// Copyright 2018 Authors of Cilium
```

Comment:
Should this file be called `daemon_options.go`? I guess it is the runtime options now, as globals.  Are we sure that we're ok making this transition, however? (from what are clearly daemon options/config to global options/config that force all components to be configured the same).